crates/bevy_color/src/lcha.rs

```diff
@@ -119,6 +121,11 @@ impl Mix for Lcha {
     }
 }
 
+impl Gray for Lcha {
+    const BLACK: Self = Self::new(0.0, 0.0, 0.0000136603785, 1.0);
+    const WHITE: Self = Self::new(1.0, 0.0, 0.0000136603785, 1.0);
```

Comment:
The third coord is a rounding error in test_colors, but I   figured it'd be better to maintain it looking just like the test data. I can round these to nearest significant digit if preferred.